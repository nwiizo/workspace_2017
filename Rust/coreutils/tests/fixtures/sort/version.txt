11.2.3
1.2.3-alpha2
1.2.3-alpha
1.12.4
