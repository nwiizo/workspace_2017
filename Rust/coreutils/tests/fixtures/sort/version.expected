1.2.3-alpha
1.2.3-alpha2
1.12.4
11.2.3
